src/httpserver.cpp

```diff
@@ -940,6 +941,92 @@ bool HTTPRequest::LoadBody(LineReader& reader)
     }
 }
 
+void HTTPRequest::WriteReply(HTTPStatusCode status, std::span<const std::byte> reply_body)
```

Comment:
Would it be possible to allow writing a reply composed of several `span<byte>`?
It should allow sending while serializing a response object.

Reply:
Sure, thanks :)